packages/alfa-rules/src/sia-r83/rule.ts

```diff
@@ -93,32 +92,41 @@ function isPossiblyClipped(device: Device): Predicate<Element> {
         .some((whiteSpace) => whiteSpace.value === "nowrap")
     ) {
       return style
-        .computed("overflow-x")
-        .some(
-          (overflow) => overflow.value === "hidden" || overflow.value === "clip"
+        .cascaded("overflow-x")
+        .some((overflow) =>
+          overflow.some(
+            (overflow) =>
+              overflow.value === "hidden" || overflow.value === "clip"
+          )
         );
     }
 
-    // Case 2: The height of the element has been restricted. In this case, text
-    // might clip if overflow of the y-axis is hidden.
+    // Case 2: The height of the element has been restricted using an non-font
+    // relative length not set via the `style` attribute. In this case,
+    // text might clip if overflow of the y-axis is hidden.
+    //
+    // For heights set via the `style` attribute, we assume that its value is
+    // controlled by JavaScript and is adjusted as the content scales.
```

Comment:
As far as I understand, font relative height are assumed to leave enough room for several lines (#546) 🤔 
That should also be recorded here (especially since it is the bit requiring to use the cascaded value rather than the computed one).